tests/app/test_jinja_filters.py

```diff
@@ -129,6 +129,28 @@ def test_format_date_month_year(self):
         # Then
         self.assertEqual(format_value, "<span class='date'>January 2017</span>")
 
+    def test_format_date_time_in_dst(self):
+        # Given
+        date_time = '2018-03-29T11:59:13.528680'
+        date_format = '%d %B %Y at %H:%M'
+
+        # When
+        format_value = format_datetime(date_time, date_format)
+
+        # Then
+        self.assertEqual(format_value, "<span class='date'>29 March 2018 at 12:59 (GMT+1)</span>")
+
+    def test_format_date_time_in_non_dst(self):
```

Comment:
If it wasn't for the fact i'm leaving another comment then I might have let this go but I don't like the use of "non_bst". This is GMT. 
`test_format_date_time_gmt`.
Above it's not really dst it's BST or GMT+1
`test_format_date_time_bst`
We're setting London time then using American terminology.